processor/src/lib.rs

```diff
@@ -128,6 +133,7 @@ pub fn execute_iter<A>(
     program: &Program,
     stack_inputs: StackInputs,
     advice_provider: A,
+    _options: ExecutionOptions,
 ) -> VmStateIterator
```

Comment:
I'm actually not sure if we need to set execution options for `execute_iter()` - let's skip this for now.